src/components/Col/Col.tsx

```diff
@@ -1,6 +1,8 @@
 import classNames from "classnames";
 import PropTypes from "prop-types";
-import React from "react";
+import React, { ElementType, ReactNode } from "react";
+
+export type ColSize = 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12;
 
 export const colSizes = [
```

Comment:
@huwshimi Not sure where it is used, but this seems to list both numbers and strings. Is that expected?

Reply:
Yeah I think that was to make it easier to not make mistakes, but now that it's typescript I think it's probably easier to just have numbers and typescript will tell you the correct type.

Reply:
I'll update this to be numbers as well so that it's consistent.